src/components/Socials/styles.scss

```
@@ -1,3 +1,83 @@
+// Mobile portrait
 .wrapper {
-	font-size: 1em; //nothing comment to make sure class appears
+	display: flex;
+	justify-content: space-between;
+	font-size: 2rem;
+	
+	&.closed {
+		opacity: 0;
+	}
+
+	&.open {
+		width: 50%;
```

```suggestion
		width: 50%;
		max-width: 10em;
```
Otherwise on desktop they grow to a really large space that isn't similar to the designs.  I'm not sure if it's `10em`, but worth giving a quick measure.

---

Went with 6em, looks better. Not sure how I missed the navOpen class name still being used 🙈